helm/ako/templates/configmap.yaml

```diff
@@ -25,6 +25,9 @@ data:
   {{ if .Values.AKOSettings.syncNamespace  }}
   syncNamespace: {{ .Values.AKOSettings.syncNamespace | quote }}
   {{ end }}
+  {{ if .Values.AKOSettings.namespaceSyncLabel  }}
+  namespaceSyncLabel: {{ .Values.AKOSettings.namespaceSyncLabel | quote }}
```

Comment:
@hemantavi : I will updated above label as namespaceSelector. Regarding format of value in values.yaml.. I feel we should use 
namespaceSelector:
label-key: ""
label-value: ""

Above same format, we have used for nodeport selector.. we should be uniform in our approach.  Only thing with this is we need to make documentation bit mode explicit about how these values should be reflected while doing labeling to namespace, otherwise customer will be confused.

@sudswasavi  what is your thoughts on this?

Reply:
yeah makes sense.